packages/shared/src/components/post/NewComment.tsx

```diff
@@ -98,41 +96,51 @@ function NewCommentComponent(
 
   if (typeof inputContent !== 'undefined') {
     return (
-      <CommentMarkdownInput
+      <CommentInputOrPage
         {...props}
         post={post}
-        className={{ container: 'my-4', tab: className?.tab }}
+        className={{ input: { container: 'my-4', tab: className?.tab } }}
         onCommented={onSuccess}
         initialContent={inputContent}
+        onClose={() => setInputContent(undefined)}
       />
     );
   }
 
+  const pictureClasses = 'hidden tablet:flex';
+
   return (
     <button
       type="button"
       className={classNames(
-        'flex w-full items-center rounded-16 border border-theme-divider-tertiary bg-theme-float p-3 typo-callout hover:border-theme-divider-primary hover:bg-theme-hover',
+        'flex w-full items-center gap-4 rounded-16 border-t border-theme-divider-tertiary bg-blur-highlight p-3 typo-callout hover:border-theme-divider-primary hover:bg-theme-hover tablet:border tablet:bg-theme-float',
         className?.container,
       )}
       onClick={() => onCommentClick(Origin.StartDiscussion)}
     >
       {user ? (
-        <ProfilePicture user={user} size={size} nativeLazyLoading />
+        <ProfilePicture
+          user={user}
+          size={size}
+          nativeLazyLoading
+          className={pictureClasses}
+        />
       ) : (
         <Image
           src={fallbackImages.avatar}
           alt="Placeholder image for anonymous user"
-          className={getProfilePictureClasses('large')}
+          className={classNames(
+            pictureClasses,
+            getProfilePictureClasses('large'),
+          )}
         />
       )}
-      <span className="ml-4 text-theme-label-tertiary">
-        Share your thoughts
-      </span>
+      <span className="text-theme-label-tertiary">Share your thoughts</span>
       <Button
         size={buttonSize[size]}
-        className="ml-auto text-theme-label-primary"
+        className="ml-auto hidden text-theme-label-primary tablet:flex"
         variant={ButtonVariant.Secondary}
+        tag="a"
```

Comment:
This should not render as a button, since it's within a button and is not clickable. So changed it to be a link.